stock_request_analytic/views/stock_request_order_views.xml

```diff
@@ -8,8 +8,16 @@
         <field name="inherit_id" ref="stock_request.stock_request_order_form" />
         <field name="arch" type="xml">
             <div name="button_box" position="inside">
-                <field name="analytic_account_ids" invisible="1" />
-                <field name="analytic_tag_ids" invisible="1" />
+                <field
+                    name="analytic_account_ids"
+                    groups="analytic.group_analytic_accounting"
```

Comment:
Putting groups here in this version doesn't avoid permission errors. It should be at view level. Anyway, removing the analytic group is only for not showing the fields, not for permissions.

Reply:
Ok, it is already changed and the corresponding groups are set in the views.